### PR TITLE
Adapted Jenkinsfile for openHAB 3

### PR DESCRIPTION
**Pull request by kaikreuzer**
Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: launch/Jenkinsfile [hunk 1]
```diff
@@ -68,7 +68,7 @@ def releaseOpenHabComponent(componentName, branch, releaseVersion, nextVersion,
                 def mvnSnapshotOptions = ""
                 def mvnOptionsMap = [:]
                 
-                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false' //
+                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false -Dspotless.check.skip=true' //
```
Comment (wborn) [inline launch/Jenkinsfile:71]: It probably also needs to be added to the releaseArgs:

```suggestion
                mvnOptionsMap["build"] = '-DreleaseArgs=skipTests=true,skipChecks=true,spotless.check.skip=true, -DskipChecks=true -DskipTests=true -DgenerateBackupPoms=false -Dspotless.check.skip=true' //
```

**Comment by kaikreuzer** [comment]
@wborn Updated.
And also removed `oh2.version` from the repo and replaced it by a `oha.version` (for openhab-addons).

File: launch/Jenkinsfile [hunk 2]
```diff
@@ -16,25 +16,25 @@ try {
...
     
     node {
         deleteDir()
-        releaseOpenHab2Component('openhab-addons', true)
-        releaseOpenHab2Component('org.openhab.binding.zwave', true)
-        releaseOpenHab2Component('org.openhab.binding.zigbee', true)
-        releaseOpenHab2Component('openhab-webui', true)
+        releaseComponent('openhab-addons', true, env.OH_BRANCH)
+        releaseComponent('org.openhab.binding.zwave', true, "master")
+        releaseComponent('org.openhab.binding.zigbee', true, "master")
```
Comment (wborn) [inline launch/Jenkinsfile:29]: This will fail unless @cdjackson updates the code to be compatible with OH3.

Comment (kaikreuzer) [reply launch/Jenkinsfile:29]: That's correct, but he planned to make the code compile within the next days, so I kept it in here.

Comment (wborn) [reply launch/Jenkinsfile:29]: OK, let's then merge it and see if all goes well until it reaches the ZigBee component. :wink:

**Review by wborn — APPROVED** [review]
Thanks!

**Pull request merged**